frontend/www/js/omegaup/components/user/Dependents.vue

```diff
@@ -25,11 +40,43 @@
 <script lang="ts">
 import { Vue, Component, Prop } from 'vue-property-decorator';
 import T from '../../lang';
+import * as ui from '../../ui';
 import { types } from '../../api_types';
 
 @Component
 export default class UserDependents extends Vue {
   @Prop() dependents!: types.UserDependentsPayload[];
+  @Prop() userVerificationDeadline!: Date | null;
+
   T = T;
+  ui = ui;
+
+  get bannerColor() {
```

Comment:
Por convención, le ponemos el tipo de retorno a las propiedades computadas
```suggestion
  get bannerColor(): string {
```
(desconozco porque no se quejó el linter)

Reply:
Esta bien